libraries/ESP8266WiFiMesh/examples/HelloMesh/HelloMesh.ino

```diff
@@ -1,50 +1,129 @@
 #include <ESP8266WiFi.h>
 #include <ESP8266WiFiMesh.h>
 
+String mesh_name = "Mesh_Node";
+
 unsigned int request_i = 0;
 unsigned int response_i = 0;
 
-String manageRequest(String request);
+String manageRequest(String request, ESP8266WiFiMesh *mesh_instance);
+transmission_status_t manageResponse(String response, ESP8266WiFiMesh *mesh_instance);
+void networkFilter(int number_of_networks, ESP8266WiFiMesh *mesh_instance);
 
 /* Create the mesh node object */
-ESP8266WiFiMesh mesh_node = ESP8266WiFiMesh(ESP.getChipId(), manageRequest);
+ESP8266WiFiMesh mesh_node = ESP8266WiFiMesh(manageRequest, manageResponse, networkFilter, "ChangeThisWiFiPassword_TODO", mesh_name, "", true);
 
 /**
-   Callback for when other nodes send you data
+   Callback for when other nodes send you a request
 
-   @request The string received from another node in the mesh
+   @param request The request string received from another node in the mesh
+   @param mesh_instance The "this" pointer of the ESP8266WiFiMesh instance that called the function.
    @returns The string to send back to the other node
 */
-String manageRequest(String request) {
+String manageRequest(String request, ESP8266WiFiMesh *mesh_instance) {
   /* Print out received message */
-  Serial.print("received: ");
+  Serial.print("Request received: ");
   Serial.println(request);
 
   /* return a string to send back */
   char response[60];
-  sprintf(response, "Hello world response #%d from Mesh_Node%d.", response_i++, ESP.getChipId());
+  sprintf(response, "Hello world response #%d from %s%s.", response_i++, mesh_instance->getMeshName().c_str(), mesh_instance->getNodeID().c_str());
```

Comment:
This string could easily be made larger than the size of response[60] by a user playing with the example. I suggest a more robust approach.